rocAL/source/meta_data/coco_meta_data_reader.cpp

```diff
@@ -137,8 +148,167 @@ void COCOMetaDataReader::print_map_contents() {
     }
 }
 
+void COCOMetaDataReader::generate_pixelwise_mask(std::string filename, RLE *rle_in) {
+    BoundingBoxCords bb_coords;
+    Labels bb_labels;
+    std::map<int, std::vector<RLE>> FromPoly;
```

Comment:
Please check and convert FromPoly also to snake case to make it consistent with other variables style used in this function